src/entity.c

```diff
@@ -2278,8 +2282,7 @@ ecs_entity_t ecs_new_component(
     }
 
     result = _ecs_new(world, world->t_component);
-    ecs_set(world, result, EcsComponent, {.size = size});
+    ecs_set(world, result, EcsComponent, {.size = (const uint32_t)size});
```

Comment:
Can you explain the need for a `const uint32_t` cast?

Reply:
Should just be a uint32_t.